pages/EC-WebLogic_help.xml

```diff
@@ -3095,14 +3145,78 @@
 		        <p>In the <b>StopApp</b> step, click the Log button to see the diagnostic information:</p> 
 						<br />
 		        <img src="../../plugins/EC-WebLogic/images/StopApp/EC-WLSStopApp4.png" />.
-							 
 
+        </div>
 
-            </div>
-		
-		<div id="StopManagedServer">
+        <div id="StopCluster"> <a name="StopCluster" id="StopCluster"></a>
+            <h2>StopCluster</h2>
+            <p>This procedures stops a cluster.</p>
+            <h3>Input</h3>
+            <ol>
+                <li>In the Main Menu, click <b>Admin</b> &gt; <b>Plugins</b> to open the Plugin Manager.</li>
```

Comment:
I thought the common use case is to use the procedure from the project, not from the plugin itself.